dashboard/test/helpers/script_levels_helper_test.rb

```diff
@@ -17,4 +22,32 @@ class ScriptLevelsHelperTest < ActionView::TestCase
     assert_equal '//test.code.org/api/hour/finish/frozen', hoc_finish_url(Script.find_by_name('frozen'))
     assert_equal '//test.code.org/api/hour/finish/course4', hoc_finish_url(Script.find_by_name('course4'))
   end
+
+  test 'script name instead of stage name in header for HOC' do
+    self.stubs(:current_user).returns(nil)
+    @script = Script.find_by_name(Script::HOC_NAME)
+    @script_level = @script.get_script_level_by_chapter 1
+    @stage = @script_level.stage
+    @game = @script_level.level.game
```

Comment:
@laurelfan I wasn't sure the best way to update these unit tests.  Should I replace `@stage` with `@script_level.stage` and `@game` with `@script_level.level.game` in the `header_progress` method?  That way I don't have to set up as many instance variables for each test.